projects/services/sidechain-connector-service-typescript/.yarnrc

```diff
@@ -1,4 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please move it back. It's there for a reason.

Reply:
yeah, i agree with @idandash its wrong, what are we missing?

Reply:
It's there because it cuts the build time and the size of the image.